components/layout/HeroBanner.tsx

```diff
@@ -1,175 +1,49 @@
-import { JSX } from '@emotion/react/jsx-runtime';
 import {
   Box,
-  Button,
+  Container,
   SxProps,
-  Theme,
   Typography,
-  TypographyProps,
 } from '@mui/material';
-import MuiMarkdown from 'mui-markdown';
 import React from 'react';
 
-import { AriaLabels } from '../../content/content.types.ts';
-import useLinkHandler from '../functional/LinkHandler.tsx';
-import ImageContainer from './ImageContainer.tsx';
-import SectionContainer from './SectionContainer.tsx';
+import EligibilityButton from '../helper/EligibilityButton.tsx';
 
 interface HeroBannerProps {
   header: string;
-  subheading?: string;
-  subheading2?: string;
-  imgsrc?: string;
-  ctaText?: string;
-  ctaLink?: string;
-  children?: JSX.Element | null;
   overrideStyles?: SxProps;
-  priority?: boolean;
-  ariaLabels?: AriaLabels;
 }
 
 const heroStyles: SxProps = {
-  backgroundColor: 'primary.dark',
+  backgroundImage: 'url(/illustrations/h1-home-desktop.png)',
+  backgroundSize: 'cover',
+  backgroundPosition: 'center',
   color: 'primary.contrastText',
+  width: '100%',
+  height: '100vh',
   py: {
     xs: 0.5,
     md: 8,
   },
 };
 
-const getHeroAdditionalsStyles = (theme: Theme) => ({
-  display: 'flex',
-  margin: theme.spacing(3, 'auto'),
-  justifyContent: 'center',
-  order: { xs: '1', md: '2' },
-});
-
 export default function HeroBanner({
   header,
-  subheading,
-  subheading2,
-  imgsrc,
-  ctaText,
-  ctaLink,
-  children = null,
   overrideStyles,
-  priority = false,
-  ariaLabels,
 }: HeroBannerProps) {
-  const { handleLinkClick } = useLinkHandler();
-
   return (
     <Box className="hero-banner" sx={overrideStyles || heroStyles} textAlign="left">
-      <SectionContainer>
-        <Typography className="hero-title" variant="h1" sx={{ display: 'block' }}>
-          {header}
-        </Typography>
-        <Box
-          className="hero-content"
+      <Container>
+        <Typography
+          className="hero-title"
+          variant="h1"
```

Comment:
The text on the hero is too wide and the margins between lines are not correct. This is probably due to me adding bottom margins in the theme file, so we may need to either adjust that or adjust in this component specifically.